pkg/hintrunner/zero/zerohint_usort_test.go

```diff
@@ -114,5 +115,166 @@ func TestZeroHintUsort(t *testing.T) {
 				},
 			},
 		},
+		"UsortBody": {
+			{
+				// input length greater then allowed size
+				operanders: []*hintOperander{
+					{Name: "input", Kind: apRelative, Value: addr(7)},
+					{Name: "input_length", Kind: apRelative, Value: feltUint64(20)},
+					{Name: "output", Kind: uninitialized},
+					{Name: "output_length", Kind: uninitialized},
+					{Name: "multiplicities", Kind: uninitialized},
+				},
+				makeHinter: func(ctx *hintTestContext) hinter.Hinter {
+					return newUsortBodyHint(ctx.operanders["input"], ctx.operanders["input_length"], ctx.operanders["output"], ctx.operanders["output_length"], ctx.operanders["multiplicites"])
+				},
+				ctxInit: func(ctx *hinter.HintRunnerContext) {
+					ctx.ScopeManager.EnterScope(map[string]any{
```

Comment:
Entering a new scope here is nice as it simulates UsortEnterScope hint which is executed right before this one 👌